be/src/io/cache/fs_file_cache_storage.cpp

```diff
@@ -309,15 +309,31 @@ std::string FSFileCacheStorage::get_path_in_local_cache(const UInt128Wrapper& va
 }
 
 Status FSFileCacheStorage::upgrade_cache_dir_if_necessary() const {
-    /// version 1.0: cache_base_path / key / offset
-    /// version 2.0: cache_base_path / key_prefix / key / offset
+    /* 
```

Comment:
add more log to show whether we need to "upgrade", print previous and current version before right after "read_file_cache_version"
add some stats like file count,  time consumption and etc when upgrade